### DISCUSSION TITLE
vendor specific IP cores

### DISCUSSION THREAD

**Discussion by chim20air** [cocotb/cocotb#3003] [Q&A]
Hi,
I was wondering how to include IP cores that are vendor specifics, something like uPROM vendor specific. I was trying to find how to compile them, and which files i need to do this kind of operations, but i couldn't find anything on this
Thanks for your time

**Comment by themperek** [comment]
It will depend on your simulator not on `cocotb`.  Just include them in `VERILOG_SOURCES` / `VHDL_SOURCES `?

**Reply by chim20air** [reply]
thanks!!!
